src/main/scala/uk/gov/homeoffice/drt/applicative/ArrivalFlights.scala

```diff
@@ -0,0 +1,25 @@
+package uk.gov.homeoffice.drt.applicative
+
+import cats.Applicative
+import cats.syntax.all._
+import uk.gov.homeoffice.drt.model.{Arrivals, RequestedFlightDetails}
+import uk.gov.homeoffice.drt.repository.ArrivalTableData
+import uk.gov.homeoffice.drt.service.ArrivalService
+
+
+trait ArrivalFlights[F[_]] {
```

Comment:
What is the `[F[_]]` syntax for here? I presume it's to do with cats, but what does it give us?

Reply:
It is related to cats-effect . Cats-effect make it very functional with no side effect . So we wrap most of class as Monaid behavior which make code to declarative rather than executing where it is declared .

Reply:
:+1: i need to read up on cats a bit more :)